.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -70,7 +66,15 @@ build:
         - from: 'out\loc'
           to: 'loc'
           include:
-            - '**/*' 
+            - '**/*'
+    - !!buildcommand
+      name: "Generate Templates"
+      command: 'scripts\generateTemplates.cmd'
+      artifacts:
+        - from: 'out\loc'
+          to: 'loc'
+          include:
+            - '**/*'
```

Comment:
i can never remember the rules here.  do *both* of these steps need to dclare they have output artifacts?  or can the first one be just a `!!defaultcommand`, and the second one be a `!!buildcommand` with outputs? you don't get 2 different copies of the outputs this way?